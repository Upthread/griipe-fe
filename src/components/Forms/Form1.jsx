import React, { Component } from "react";
import { BrowserRouter as Router, Link, Route, Switch } from "react-router-dom";
import MuiThemeProvider from "material-ui/styles/MuiThemeProvider";
import AppBar from "material-ui/AppBar";
import TextField from "material-ui/TextField";
import RaisedButton from "material-ui/RaisedButton";
import PlacesAutocomplete from 'react-places-autocomplete';
import LocationSearch from './LocationSearchInput';



class Form1 extends Component {
  continue = e => {
    e.preventDefault();
    this.props.nextStep();
  };

  back = e => {
    e.preventDefault();
    this.props.prevStep();
  };

  

  



  render() {
    
    
    return (
      <MuiThemeProvider>
        <>
          <h1>Form1</h1>
            <LocationSearch />
          <br />
          <RaisedButton
            label="Continue"
            primary={true}
            style={styles.button}
            onClick={this.continue}
          />
        </>
      </MuiThemeProvider>
    );
  }
}

const styles = {
  button: {
    margin: 15
  }
};

export default Form1;
