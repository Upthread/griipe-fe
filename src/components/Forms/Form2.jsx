import React, { Component } from "react";
import { BrowserRouter as Router, Link, Route, Switch } from "react-router-dom";
import MuiThemeProvider from "material-ui/styles/MuiThemeProvider";
import AppBar from "material-ui/AppBar";
import TextField from "material-ui/TextField";
import RaisedButton from "material-ui/RaisedButton";

class Form2 extends Component {
  continue = e => {
    e.preventDefault();
    this.props.nextStep();
  };

  back = e => {
    e.preventDefault();
    this.props.prevStep();
  };
  state = {};

  render() {
    const { values, handleChange } = this.props;
    return (
      <MuiThemeProvider>
        <>
          <h1>Record Complaint</h1>

          
          <br />

          <RaisedButton
            label="Back"
            primary={false}
            style={styles.button}
            onClick={this.back}
          />

          <RaisedButton
            label="Continue"
            primary={true}
            style={styles.button}
            onClick={this.continue}
          />


        </>
      </MuiThemeProvider>
    );
  }
}

const styles = {
  button: {
    margin: 15
  }
};

export default Form2;
