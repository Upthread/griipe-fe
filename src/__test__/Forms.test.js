import React from "react";
import test from "react-test-renderer";

import Form1 from "../components/Forms/Form1"
import Form2 from "../components/Forms/Form2"

describe("<Form1 />", () => {
  it("matches snapshot", () => {
    const tree = test.create(<Form1 />);

    expect(tree.toJSON()).toMatchSnapshot();
  });
});

describe("<Form2 />", () => {
  it("matches snapshot", () => {
    const tree = test.create(<Form2 />);

    expect(tree.toJSON()).toMatchSnapshot();
  });
});
