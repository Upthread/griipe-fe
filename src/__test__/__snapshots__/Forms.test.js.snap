// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Form1 /> matches snapshot 1`] = `
<div
  className="recording-loader loader"
>
  <h1>
    Griipe
  </h1>
  <br />
  <div
    className="spinner-border"
    role="status"
    style={
      Object {
        "height": "3rem",
        "width": "3rem",
      }
    }
  >
    <span
      className="sr-only"
    >
      Loading...
    </span>
  </div>
</div>
`;

exports[`<Form2 /> matches snapshot 1`] = `
<div
  className="recording-loader loader"
>
  <h1>
    Griipe
  </h1>
  <br />
  <div
    className="spinner-border"
    role="status"
    style={
      Object {
        "height": "3rem",
        "width": "3rem",
      }
    }
  >
    <span
      className="sr-only"
    >
      Loading...
    </span>
  </div>
</div>
`;
